docs/integrations/gitlab/discovery.md

```diff
@@ -6,14 +6,58 @@ sidebar_label: Discovery
 description: Automatically discovering catalog entities from repositories in GitLab
 ---
 
-The GitLab integration has a special discovery processor for discovering catalog
-entities from GitLab. The processor will crawl the GitLab instance and register
-entities matching the configured path. This can be useful as an alternative to
+The GitLab integration has a special entity provider for discovering catalog
+entities from GitLab. The entity provider will crawl the GitLab instance and register
+entities matching the configured paths. This can be useful as an alternative to
 static locations or manually adding things to the catalog.
 
-To use the discovery processor, you'll need a GitLab integration
-[set up](locations.md) with a `token`. Then you can add a location target to the
-catalog configuration:
+To use the discovery provider, you'll need a GitLab integration
+[set up](locations.md) with a `token`. Then you can add a provider config per group
```

Comment:
no support for going through ALL groups? maybe that doesn't make sense mostly?

Reply:
I think it would be nice. Especially being able to specify just a group and all of the subgroups automatically just work as well

Reply:
Okay, I will also try to add that later. Makes sense and makes everything easier as well